### PR TITLE
feat: features

### PR DESCRIPTION
**Pull request by livio-a**

### CODE REVIEW

File: internal/admin/repository/eventsourcing/eventstore/features.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+	for _, event := range events {
+		if err := featuresCopy.AppendEvent(event); err != nil {
+			return model.FeaturesToModel(&featuresCopy), nil
+		}
+	}
+	return model.FeaturesToModel(&featuresCopy), nil
+}
+
+func (repo *FeaturesRepo) GetOrgFeatures(ctx context.Context, orgID string) (*features_model.FeaturesView, error) {
+	features, err := repo.View.FeaturesByAggregateID(orgID)
```
Comment (fgerschwiler) [inline internal/admin/repository/eventsourcing/eventstore/features.go:54]: do we need to read the missing events from eventstore here?

File: internal/authz/repository/eventsourcing/eventstore/token_verifier.go [hunk 2]
```diff
@@ -150,3 +209,36 @@ func (u *TokenVerifierRepo) getIAMByID(ctx context.Context) (*iam_model.IAM, err
...
+		logging.Log("EVENT-PSoc3").WithError(esErr).Debug("error retrieving new events")
+		return nil
+	}
+	featuresCopy := *features
+	for _, event := range events {
+		if err := featuresCopy.AppendEvent(event); err != nil {
+			return nil
+		}
+	}
+	return nil
```
Comment (fgerschwiler) [inline internal/authz/repository/eventsourcing/eventstore/token_verifier.go:235]: i think checkFeatures is missing here

File: internal/features/repository/view/features_view.go [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+		f[i] = features
+	}
+	return save(db, f...)
+}
+
+//
+//func DeleteOrg(db *gorm.DB, table, orgID string) error {
+//	delete := repository.PrepareDeleteByKey(table, model.OrgSearchKey(org_model.OrgSearchKeyOrgID), orgID)
+//	return delete(db)
+//}
```
Comment (fgerschwiler) [inline internal/features/repository/view/features_view.go:54]: remove if not used

**Review by fgerschwiler — COMMENTED** [review]
I think we need a request where we return the features as strings for frontend

File: internal/admin/repository/eventsourcing/eventstore/features.go (hunk 1 above)
Comment (livio-a) [reply internal/admin/repository/eventsourcing/eventstore/features.go:54]: as discussed... we leave it for now

File: internal/authz/repository/eventsourcing/eventstore/token_verifier.go (hunk 2 above)
Comment (livio-a) [reply internal/authz/repository/eventsourcing/eventstore/token_verifier.go]: done

**Review by fgerschwiler — APPROVED** [review]

**Review by fgerschwiler — APPROVED** [review]

**Review by fgerschwiler — APPROVED** [review]

**Pull request merged**